src/ProductsXmlFeed.php

```diff
@@ -312,7 +312,7 @@ private static function get_property_g_additional_image_link( $product, $propert
 
 		if ( $attachment_ids && $product->get_image_id() ) {
 			foreach ( $attachment_ids as $attachment_id ) {
-				$images[] = wp_get_attachment_image_src( $attachment_id )[0];
+				$images[] = wp_get_attachment_image_src( $attachment_id, 'full' )[0];
```

Comment:
I think we should use `woocommerce_single` as we also do for the [main image](https://github.com/saucal/pinterest-for-woocommerce/blob/ae5620b478571ca573b0bc95f5ecc29449812c24/src/ProductsXmlFeed.php#L201).

Full means unrestricted which can be huge and potentially cause issues.

Reply:
Done